service/matching/matchingEngine_test.go

```diff
@@ -2005,44 +2006,185 @@ func (s *matchingEngineSuite) TestGetVersioningData() {
 	s.Equal("5", majorSets[0].GetBuildIds()[0])
 }
 
-func (s *matchingEngineSuite) TestActivityQueueMetadataInvalidate() {
-	// Overwrite the matching mock - we expect one and only one fetch call here, after the activity queue is invalidated
-	mockMatch := matchingservicemock.NewMockMatchingServiceClient(s.controller)
-	mockMatch.EXPECT().GetTaskQueueUserData(gomock.Any(), gomock.Any()).
-		Return(&matchingservice.GetTaskQueueUserDataResponse{}, nil).
-		Times(1)
-	s.matchingEngine.matchingClient = mockMatch
+func (s *matchingEngineSuite) TestGetTaskQueueUserData_NoData() {
+	namespaceID := namespace.ID(uuid.New())
+	tq := "makeToast"
+
+	res, err := s.matchingEngine.GetTaskQueueUserData(context.Background(), &matchingservice.GetTaskQueueUserDataRequest{
+		NamespaceId:              namespaceID.String(),
+		TaskQueue:                tq,
+		LastKnownUserDataVersion: 0,
+	})
+	s.NoError(err)
+	s.False(res.TaskQueueHasUserData)
+	s.Nil(res.UserData.GetData())
+}
 
+func (s *matchingEngineSuite) TestGetTaskQueueUserData_ReturnsData() {
 	namespaceID := namespace.ID(uuid.New())
-	tq := "tupac"
+	tq := "makeToast"
```

Comment:
I'm insulted

Reply:
Hah, I can put it back. That test got deleted and I wrote more by copying and pasting bits of other tests so it's kind of an artifact of the diff